packages/genesys-spark-components/src/components/stable/gux-form-field/functional-components/gux-form-field-legend-label-container/gux-form-field-legend-label-container.tsx

```diff
@@ -0,0 +1,22 @@
+import { FunctionalComponent, h, VNode } from '@stencil/core';
+
+import { GuxFormFieldLabelPosition } from '../../gux-form-field.types';
+
+interface GuxFormFieldLegendLabelContainerProps {
+  labelPosition: GuxFormFieldLabelPosition;
+}
+
+export const GuxFormFieldLegendLabelContainer: FunctionalComponent<
+  GuxFormFieldLegendLabelContainerProps
+> = ({ labelPosition }, children): VNode => {
+  return (
+    <div
+      class={{
+        'gux-form-field-legend-label-container': true,
+        [`gux-${labelPosition}`]: true
+      }}
+    >
+      {children}
+    </div>
+  ) as VNode;
+};
```

Comment:
@daragh-king-genesys I think this is needed because of the way the GuxFormFieldLegendLabel is set up. Can't add the tooltip directly to that component as it will be hidden. 

What are your thoughts on the name here? It's probably a bit long. Would GuxLegendLabelContainer be ok?